web-registry/src/components/organisms/MediaForm/MediaFormLegacy.tsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import { useTheme } from '@mui/styles';
+import {
+  useMediaQuery,
+  Grid,
+  FormHelperText,
+  styled,
+  Box,
+} from '@mui/material';
+import { Form, Field, getIn, useFormikContext } from 'formik';
+import { useParams } from 'react-router-dom';
+
+import { ImageUpload } from 'web-components/lib/components/inputs/ImageUpload';
+import { VideoInput } from 'web-components/lib/components/inputs/VideoInput';
+import FormLabel from 'web-components/lib/components/inputs/FormLabel';
+import { UrlType } from 'web-components/lib/utils/schemaURL';
+
+import getApiUri from '../../../lib/apiUri';
+import { useMediaFormStyles } from './useMediaFormStyles';
+
+import { cropAspect, MediaBaseValues, MediaBaseErrors } from './MediaForm';
+
+export interface MediaValuesLegacy extends MediaBaseValues {
+  'regen:landStewardPhoto'?: UrlType;
+}
+
+type valueObject = { '@value': string };
+export interface MediaErrorsLegacy extends MediaBaseErrors {
+  'regen:galleryPhotos'?: string;
+  'regen:landStewardPhoto'?: valueObject;
+}
+
+const GalleryImgGrid = styled(Grid)(({ theme }) => ({
+  [theme.breakpoints.up('sm')]: {
+    height: theme.typography.pxToRem(169),
+    flex: 1,
+  },
+  [theme.breakpoints.down('sm')]: {
+    height: theme.typography.pxToRem(139),
+  },
+}));
+
+/** Form content for legacy projects */
+const MediaFormLegacy = (): JSX.Element => {
+  const styles = useMediaFormStyles();
+  const theme = useTheme();
+  const apiUri = getApiUri();
+  const { projectId } = useParams();
+  const isTabletOrLarger = useMediaQuery(theme.breakpoints.up('sm'));
+
+  const ImageFieldFull = ({
+    buttonText = '+ Add Photo',
+    ...props
+  }: {
+    buttonText?: string;
+    description?: string;
+    label?: string;
+    labelSubText?: string;
+    name: string;
+  }): JSX.Element => (
+    <Field
+      apiServerUrl={apiUri}
+      buttonText={buttonText}
+      classes={{ main: styles.fullSizeMedia }}
+      component={ImageUpload}
+      description={props.description}
+      fixedCrop={cropAspect}
+      label={props.label}
+      labelSubText={props.labelSubText}
+      name={props.name}
+      projectId={projectId}
+      optional
+      isDrop
+    />
+  );
+
+  const ImageFieldSmall = (props: {
+    buttonText?: string;
+    hideDragText?: boolean;
+    name: string;
+  }): JSX.Element => (
+    <Field
+      apiServerUrl={apiUri}
+      buttonText={props.buttonText}
+      classes={{ button: styles.smallButton }}
+      component={ImageUpload}
+      fixedCrop={cropAspect}
+      hideDragText={props.hideDragText}
+      name={props.name}
+      projectId={projectId}
+      isDrop
+    />
+  );
+
+  const { errors, touched } = useFormikContext<MediaValuesLegacy>();
+
+  return (
+    <Form translate="yes">
+      <ImageFieldFull
```

Comment:
it seems like the errors are not propagated to this component

Reply:
I made this to DRY up the code, but you're right - good catch. The errors are passed by formik's `Field` component, which this structure blocks. I extracted the shared props to a separate object and that seemed to fix